### ISSUE TITLE
Property 'checkboxable' does not exist on type 'TableColumn'

### ISSUE THREAD

**Issue by m0squito** [swimlane/ngx-datatable#664]
**I'm submitting a ...**  (check one with "x")
```
[x ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here
```

**Current behavior**

ng serve --prod fails to compile with the following error:

> node_modules/@swimlane/ngx-datatable/release/components/body/body-cell.component.ngfactory.ts (1,1): Property 'checkboxable' does not exist on type 'TableColumn'

**Expected behavior**

ng serve --prod does not fail to compile and run

**Reproduction of the problem**

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

macOS 10.12.4

* **Table version:** 0.7.x

7.2.0

* **Angular version:** 2.0.x

4.0.1

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

Chrome 57.0.2987.133 (64-bit)

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
TypeScript 2.2.2

**Comment by adammedford** [comment]
Property is missing from https://github.com/swimlane/ngx-datatable/blob/master/src/types/table-column.type.ts

**Comment by amcdnl** [comment]
Fixed in 7.2.1 :)
